core/src/test/scala/integration/kafka/api/CustomQuotaCallbackTest.scala

```diff
@@ -87,7 +87,6 @@ class CustomQuotaCallbackTest extends IntegrationTestHarness with SaslSetup {
 
   override def configureSecurityBeforeServersStart(testInfo: TestInfo): Unit = {
     super.configureSecurityBeforeServersStart(testInfo)
-    createScramCredentials("", JaasTestUtils.KAFKA_SCRAM_ADMIN, JaasTestUtils.KAFKA_SCRAM_ADMIN_PASSWORD)
```

Comment:
This class is currently disabled but should we call the other `createScramCredentials()` with the `Admin` argument or leave a comment so it's easier to re-enable later on?

Reply:
Thanks for pointing out.
I have replaced it with `createScramCredentials(Admin, String, String)`